django/thunderstore/api/views/community.py

```diff
@@ -0,0 +1,72 @@
+from django.db.models import Count, Sum
+from django.http import HttpRequest, HttpResponse
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from thunderstore.api.serializers import CyberstormCommunitySerializer
+from thunderstore.community.models import Community
+
+
+class CommunityAPIView(APIView):
+    permission_classes = []
+
+    @swagger_auto_schema(
+        responses={200: CyberstormCommunitySerializer()},
+        operation_id="api.community",
+    )
+    def get(self, request: HttpRequest, community_identifier: str) -> HttpResponse:
+        # TODO: Filter out the common shared packages somehow
+        c = Community.objects.annotate(
+            pkgs=Count("package_listings", distinct=True),
+            downloads=Sum(
+                "package_listings__package__versions__downloads", distinct=True
+            ),
+        ).get(
+            identifier=community_identifier,
+        )
+        community = {
+            "name": c.name,
+            "namespace": c.identifier,
+            "downloadCount": c.downloads,
+            "packageCount": c.pkgs,
+            "imageSource": c.icon.url if bool(c.icon) else None,
```

Comment:
Hmm, good point. We might have to add the community "card images" to TS, so that Cyberstorm can have those also. I'll change this to the background image, since it was used somewhere at least in Cyberstorm. I'll make a separate task out of adding the "card image" field for the community model.